### PR TITLE
Fix firstPair issue #452

### LINKED ISSUE #452: Fix findPair method.
There is currently a method called `findPair` that lives in `/lib/findPair.js`.
It is incomplete and needs to be fixed!

Given a desired sum and an array of integers, findPair returns the first set of two array elements that, if added together, would result in the desired sum.

For example findPair(7, [1, 4, 5, 3, 6]) would be expected to return [4, 3]

### Acceptance Criteria:
1. Running `yarn test findPair` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by josephbhunt**
Closes #452 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

This was way harder than I thought it would be.

### CODE REVIEW

File: test/areaOfCircleSector.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+describe("areaOfCircleSector", () => {
+  test("throws an error for invalid types", () => {
+    expect(() => areaOfCircleSector("pizza", 60)).toThrow("Invalid Argument");
+    expect(() => areaOfCircleSector(5, "pizza")).toThrow("Invalid Argument");
+  });
+
+  test("returns the area of the sector", () => {
+    expect(areaOfCircleSector(5, 60)).toBe(13.09);
+  });
```
Comment (ktilcu) [inline test/areaOfCircleSector.test.js]: Are there lower and upper bounds to sectors?

Comment (josephbhunt) [reply test/areaOfCircleSector.test.js]: Good catch. Will fix.

Comment (ktilcu) [reply test/areaOfCircleSector.test.js]: 👍

Comment (josephbhunt) [reply test/areaOfCircleSector.test.js]: @ktilcu I added the requested tests. Something happened the last time I synched with upstream and commits got duplicated. Not sure what that's all about.

**Comment by BlakeGuilloud** [comment]
Hey @josephbhunt ! It looks like you are trying to commit changes to 49 different files. I'm not sure how that happened, but if you could correct it, please feel free to open a new PR. Thanks!

**Pull request closed without merging**